libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -94,6 +95,11 @@ void SRV_Channel::output_ch(void)
             }
         }
     }
+    #else 
+        #define _UNUSED(x) ((void)(x))
```

Comment:
```suggestion
        #define UNUSED_LOCAL(x) ((void)(x))
```

... and move this define into AP_Common.h